### PR TITLE
Basic for

### PR DESCRIPTION
**Pull request by himanshuc3**
This is the basic travis template build. Which other options do you want. I would be happy to implement and learn. #141

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "iojs"
```
Comment (aayusharora) [inline .travis.yml]: Hi @himanshuc3 

You need to setup before_script , script, before_install and cache options 
Use https://docs.travis-ci.com/user/languages/javascript-with-nodejs/ for reference

Thanks

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "stable"
+script: gulp travis
```
Comment (abhishek97) [inline .travis.yml]: why? we don't even have gulp as dependency here.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "stable"
+script: gulp travis  
+before_install: npm install -g grunt-cli
```
Comment (abhishek97) [inline .travis.yml]: why is this required

File: examples/public/stylesheets/style.css [hunk 4]
```diff
@@ -3115,8 +3115,8 @@ html {
     margin-top: 25px; }
 
 .grid-background {
-  background-color: #f6f6f6;
-  padding: 5%; }
+   background-color: #f6f6f6;
```
Comment (abhishek97) [inline examples/public/stylesheets/style.css]: Please remove irrelevant changes.

File: package.json [hunk 5]
```diff
@@ -20,7 +20,7 @@
     "serve-static": "^1.13.2"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "grunt build",
```
Comment (abhishek97) [inline package.json]: script name `test` here is misleading.

**Comment by abhishek97** [comment]
@himanshuc3 you can test your `.travis.yml` for your fork.
